files/demo/https_certificate.pem

```diff
@@ -60,3 +60,4 @@ X4Po1QYz+3dszkDqMp4fklxBwXRsW10KXzPMTZ+sOPAveyxindmjkW8lGy+QsRlG
 PfZ+G6Z6h7mjem0Y+iWlkYcV4PIWL1iwBi8saCbGS5jN2p8M+X+Q7UNKEkROb3N6
 KOqkqm57TH2H3eDJAkSnh6/DNFu0Qg==
 -----END CERTIFICATE-----
+
```

Comment:
Any danger of this new line at the end causing a problem?

Reply:
Would normally think the `END CERTIFICATE` armoring should make this robust. To cross-check, the `unstable` environment has now [got the additional newline](https://github.com/libero/environments/blob/master/files/unstable/https_certificate.pem#L63) and is [still working](https://unstable.libero.pub/).